### PR TITLE
Improving types

### PR DESCRIPTION
**Pull request by PabloRosales**
Hey, adding some improvements on types for #217

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -1516,6 +1517,7 @@ declare namespace Shopify {
         width: number;
         height: number;
         updated_at: string;
+        alt: string | null;
```
Comment (lpinca) [inline index.d.ts]: This is not documented, https://help.shopify.com/en/api/reference/products/product_image is it available?

Comment (PabloRosales) [reply index.d.ts]: It is as far as I can see in all responses (shown in documentation) involving images, did some tests and I did get them when available and as null when not set.

Comment (lpinca) [reply index.d.ts]: Ok, thank you.

**Comment by lpinca** [comment]
Thank you.

**Comment by PabloRosales** [comment]
Thank you @lpinca for all the work here.

**Pull request merged**